mlflow/ml-package-versions.yml

```diff
@@ -33,7 +33,7 @@ pytorch:
 
   models:
     minimum: "1.9.0"
-    maximum: "2.1.1"
+    maximum: "2.1.2"
```

Comment:
Running the update here was just for testing purposes, right? 
For the final PR, we should just add the autologging support version ranges for the current released supported version for langchain and let our CD pipeline handle updating the package version files :)

Reply:
Yes will revert this change when merge

Reply:
If necessary, you can manually upgrade the langchain versions.